server/accept.go

```diff
@@ -0,0 +1,40 @@
+package server
+
+import (
+	"sort"
+	"strconv"
+	"strings"
+)
+
+// AcceptEntry represents a parsed Accept header entry with q-weight.
+type AcceptEntry struct {
+	MediaType string
+	QValue    float64
+}
+
+// ParseAcceptHeader parses and sorts the Accept header by q-values.
+func ParseAcceptHeader(header string) []AcceptEntry {
```

Comment:
if you need some [test cases](https://github.com/ChainSafe/lodestar/blob/unstable/packages/api/test/unit/utils/headers.test.ts#L6)

Reply:
> If there is no accept header or the accept header isn't in the supported media types, we will default to JSON.

we do the same although would return 406 if client only accepts media types we don't support, it's handled [here](https://github.com/ChainSafe/lodestar/blob/f8acf8b56f7c4e96f27e1734868733835153e98d/packages/api/src/utils/server/handler.ts#L42-L57)

Reply:
Hmm yes. I like that idea. Will do the same.